assets/demos/DemoApp.Tests.ps1

```diff
@@ -0,0 +1,154 @@
+BeforeAll {
+  Import-Module "../../BenchPress/Helpers/Azure/BenchPress.Azure.psd1"
+}
+
+Describe 'Service Plan Tests' {
+  it 'Should contain a service plan with the given name' {
+    #arrange
+    $rgName = "rg-${env:ENVIRONMENT_NAME}"
+    $svcPlanName = "plan-${env:ENVIRONMENT_SUFFIX}"
+
+    #act
+    $exists = Get-AzBPAppServicePlan -ResourceGroupName $rgName -AppServicePlanName $svcPlanName
```

Comment:
`exists` variable name changed to something more aligned with the object returned like `appSvcPlan`?

Reply:
good catch! renamed the variable 👍